05-html-css/README.md

```diff
@@ -0,0 +1,18 @@
+## HTML & CSS
+
+1. [Intro to the Web](01-the-web.md)
+1. [HTML Intro](02-html-intro.md)
+1. [Content Hierarchy](04-content-hierarchy.md)
+
+1. [CSS Intro](03-css-intro.md)
+1. [Attributes, Classes & IDs](attributes-classes-ids.md)
+1. [CSS Selectors](06-css-selectors.md)
+1. [Cascade & Inheritance](09-cascade-inheritance.md)
+1. [Worksheet: Specificity](09a-specificity-practice.md)
+1. [Dev Tools](10-dev-tools-into-default-styles.md)
+
+1. [Box Model](11-box-model.md)
+1. CSS Layout
+    - [Display](12-layout-display.md)
+    - [Floats](13-layout-floats.md)
+    - [Positioning](14-layout-positioning.md)
```

Comment:
There are a bunch of `00-whatever` files in this directory, some of which have similar titles to other lectures. Can we get rid of some?